app/workers/field_tests/prune_old_experiments_worker.rb

```diff
@@ -1,13 +0,0 @@
-module FieldTests
-  class PruneOldExperimentsWorker
```

Comment:
What was the reason we had this before?

Reply:
The idea is that we would take people out of the system after a while and reintroduce them into the pool.

So if I was on feed variation A, and the experiment is meant to be long-lived, I can be reintroduced as a test subject for further evaluation to get more "modern" results. I.e. if lots of things had changed since my results were first accounted for, after a while I should just be reintroduced as a "fresh" subject with a new variation. This is only relevant if we're keeping experiments going for longer— And is really only feasible if our team is set up to administer these ongoing trials.